<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoModeTypeDialog</class>
 <widget class="QWidget" name="AutoModeTypeDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>90</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Auto Mode</string>
  </property>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>10</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>50</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="inSavingOrderRadioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>in saving order</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="byHorizontalCoordinateRadioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>by horizontal coordinate</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
